### PR TITLE
Adding one filter for all the events

### PR DESCRIPTION
**Pull request by agatsoh**
Creating this pull request which fixes https://github.com/raiden-network/raiden/issues/294

### CODE REVIEW

**Comment by LefterisJP** [comment]
Will review now. Rebased it on top of latest master.

File: raiden/network/rpc/client.py [hunk 1]
```diff
@@ -920,64 +925,15 @@ def settle(self):
...
-
-        return Filter(
-            self.client,
-            filter_id_raw,
-        )
-
-    def channelsecretrevealed_filter(self):
-        """ Install a new filter for ChannelSecret events.
+    def filter_for_all_events(self):
+        """ Install a new filter for the current netting channel contract.
```
Comment (LefterisJP) [inline raiden/network/rpc/client.py]: Install a new filter **for all the events emitted by** the current netting channel contract

File: raiden/network/rpc/client.py [hunk 2]
```diff
@@ -920,64 +925,15 @@ def settle(self):
...
-        filter_id_raw = new_filter(self.client, netting_channel_address_bin, topics)
-
-        return Filter(
-            self.client,
-            filter_id_raw,
-        )
-
-    def channelsecretrevealed_filter(self):
-        """ Install a new filter for ChannelSecret events.
+    def filter_for_all_events(self):
```
Comment (LefterisJP) [inline raiden/network/rpc/client.py]: not really a requirement, but I like the naming as it was before, and to be consistent with that we could change: 
`filter_for_all_events` -> `all_events_filter`.

Comment (agatsoh) [inline raiden/network/rpc/client.py]: Ok will make the changes and revert back.

Comment (LefterisJP) [inline raiden/network/rpc/client.py]: > revert back.

Revert what???

**Pull request merged**